features/step_definitions/basic_steps.rb

```diff
@@ -18,8 +22,8 @@
   expect(page.current_path).to eq page_path_from(page_name)
 end
 
-Then("I should be redirected to the index page") do
-  expect(page.current_path).to eq root_path
+When("I click {string} button") do |button|
+  click_button button
```

Comment:
Do we need a button here? Wouldn't it be sufficient with a link?

Reply:
both for the log in and and sign up it's a button in the log in and sign up pages